core/src/main/resources/hudson/PluginManager/advanced.properties

```diff
@@ -21,4 +21,4 @@
 # THE SOFTWARE.
 
 uploadtext=\
-  You can upload a .hpi file to install a plugin from outside the central plugin repository.
+  You can upload an .hpi file to install a plugin from outside the central plugin repository.
```

Comment:
if this correct? I would say `a`?

Reply:
The file extension would be pronounced _age-π-eye_, and since it's _an age-old question_, it's also _an .hpi file_.

(Obviously if you pronounce the period it does not apply, but I wouldn't, and I guess @jsoref wouldn't either.)

Reply:
Correct, I don't pronounce the dot, nor did the person I surveyed on IRC.